### ISSUE TITLE
Allow creating empty list without a CSV file import

### ISSUE THREAD

**Issue by hsribei** [freeCodeCamp/mail-for-good#184]
Feature request.

**Comment by karuppiah7890** [comment]
I am not sure if this will be taken up, because we don't want the user to simply create empty lists because it's more like a mess if nothing is added to it later.

**Comment by hsribei** [comment]
The user has to be able to start from somewhere if they don't already have a list on some other service. For FCC the use case was migrating from Mailchimp (or some other email service). But for some users it will be starting a new list from scratch (like me).

If they're just starting their list, there won't be a CSV file to import (like it is my case). Requiring a dummy one is confusing and unnecessary.

As far as MfG as a Mailchimp replacement goes, Mailchimp does allow one to create an empty list, then add the opt-in form to a page to get the first signups, so I think this also makes sense for MfG.

**Comment by karuppiah7890** [comment]
@hsribei I agree that you need to create an empty list when you don't have any subscribers, but the thing is, there's also a technical catch to it which I just thought about. When you create a list, you need to say what data (fields) you are going to store about the subscriber other than the Email ID. Those custom fields (eg. Name, City etc ) can be used in the Email Templates to customize and make the Email look more personalized. But all this is possible only if a CSV is imported, as it's the CSV that tells the custom fields, as it has the first row (row header) to mention the columns/fields. 

For now, we don't have a feature to add custom fields after creating the list. So, if that's done, then we can add this feature.

**Comment by deadcoder0904** [comment]
@karuppiah7890 So does this mean, Email Subscribers & Unsubscribers are managed by Mail For Good❓

I am just starting out with no subscribers & would love to use Mail For Good. So I was handling a way to manage Emails both the subscribers & unsubscribers, if its already built-in then that might be dumb thing to do 😂

So do I have to do something like 

# emails.csv

```csv
no, email
1, angie@gmail.com
2, jenny@gmail.com
```

to start out & then rest is managed by Mail For Good or do I have to manage my own subscribers & Mail For Good just handles the sending of Emails❓

**Comment by deadcoder0904** [comment]
cc @zhakkarn as you're more active here 😉

**Comment by deadcoder0904** [comment]
Okay I got it. I just created a sample csv file like

```csv
email
angie@gmail.com
```

& imported it & now everything is handled by Mail For Good :)
